music21/musicxml/xmlToM21.py

```diff
@@ -4221,9 +4271,12 @@ def xmlDirectionTypeToSpanners(
                         sp.endHeight = float(height)
                     sp.lineType = mxObj.get('line-type')
 
-                stop = spanner.SpannerAnchor()
-                self.insertCoreAndRef(totalOffset, staffKey, stop)
-                sp.addSpannedElements(stop)
+                if targetLast is not None and offsetAfterLast == totalOffset:
+                    sp.addSpannedElements(targetLast)
+                else:
+                    stop = spanner.SpannerAnchor()
```

Comment:
Not for this PR, but should SpannerAnchor have an (optional) `type='start|stop'` so when iterating we can filter on them?

Reply:
Maybe?  Seems useful, but I worry about it triggering a "person with two watches doesn't know what time it is" problem, if a SpannerAnchor is marked as 'start', but it's the last element in the spanner.